app/controllers/import/rows_controller.rb

```diff
@@ -0,0 +1,29 @@
+class Import::RowsController < ApplicationController
+  before_action :set_import
+  before_action :set_row_data, only: %i[update]
+
+  def update
+    @row.assign_attributes(row_params)
+    @row.save(validate: false)
+
+    respond_to do |format|
+      format.html { redirect_to clean_import_path(@import) }
+      format.turbo_stream { render "imports/rows/update" }
```

Comment:
I'm a bit hesitant to rely solely on Turbo streams to change the behavior of the update here.  I think we're putting too much responsibility in the stream itself and veering away from the [Turbo conventions](https://turbo.hotwired.dev/handbook/streams#progressively-enhance-when-necessary):

> It’s good practice to start your interaction design without Turbo Streams. Make the entire application work as it would if Turbo Streams were not available, then layer them on as a level-up.

Alternatively, what I think we could do is:

- Create an endpoint that returns an individual cell's data 
- Each cell in the form will be its own turbo frame
- On update, we redirect to the controller endpoint that renders the individual cell data

I think this will allow us to reduce the amount of code here and create a more straightforward flow.